docker/kubernetes-tentacle/scripts/configure-tentacle.sh

```diff
@@ -0,0 +1,211 @@
+#!/bin/bash
+set -e
+
+if [[ "$ACCEPT_EULA" != "Y" ]]; then
+    echo "ERROR: You must accept the EULA at https://octopus.com/company/legal by passing an environment variable 'ACCEPT_EULA=Y'"
+    exit 1
+fi
+
+# Tentacle Docker images only support once instance per container. Running multiple instances can be achieved by running multiple containers.
+instanceName=Tentacle
+configurationDirectory=/etc/octopus
+applicationsDirectory=/home/Octopus/Applications
+internalListeningPort=10933
+
+mkdir -p $configurationDirectory
+mkdir -p $applicationsDirectory
+
+if [ ! -f /usr/bin/tentacle ]; then
+    ln -s /opt/octopus/tentacle/Tentacle /usr/bin/tentacle
+fi
+
+function getPublicHostName() {
+    if [[ "$PublicHostNameConfiguration" == "PublicIp" ]]; then
+        curl https://api.ipify.org/
+    elif [[ "$PublicHostNameConfiguration" == "FQDN" ]]; then
+        hostname --fqdn
+    elif [[ "$PublicHostNameConfiguration" == "ComputerName" ]]; then
+        hostname
+    else
+        echo $CustomPublicHostName
+    fi
+}
+
+function validateVariables() {
```

Comment:
Because I know we are running as a kubernetes tentacle, I was able to clean up a lot of the validation here